grails-app/services/io/xh/toolbox/app/RecallsService.groovy

```diff
@@ -7,7 +7,17 @@ class RecallsService extends BaseService {
 
     def configService
 
+    //last time we tried to fetch data, did we successfully connect to the server?
```

Comment:
commenting style  is to have a space after the //.  

also, these comments are probably superfluous -- will discuss our philosophy here in person